serde-generate/runtime/typescript/serde/binaryDeserializer.ts

```diff
@@ -64,19 +65,19 @@ export abstract class BinaryDeserializer implements Deserializer {
   }
 
   public deserializeU64(): BigInt {
-    const left = this.deserializeU32();
-    const right = this.deserializeU32();
+    const high = this.deserializeU32();
+    const low = this.deserializeU32();
```

Comment:
@danprinz I think you meant the other way around: the first (little-endian-)serialized bytes are called `low`.

Reply:
ok. changed.